taskfiles/utils.yml

```diff
@@ -103,3 +103,35 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  # ===
+  # clang UTILS
+  # ===
+  clang-format:
```

Comment:
In both clang-format and clang-tidy, can we add support for passing in a list of directories rather than a single source directory? You can look at how EXCLUDE_PATHS is implemented in `compute-checksum`.